doc/getting_started/installation_linux.rst

```diff
@@ -69,8 +69,8 @@ Install additional packages required for development with Zephyr::
 
    $ pip3 install --user -r scripts/requirements.txt
 
-Note that CMake version 3.8.2+ or higher is required. If you are using a
-distribution that currently ships an older version, check the backports or
+CMake version 3.8.2+ or higher is required. Check what version you have using
```

Comment:
Please take chance to remove "+" from "3.8.2+". It should be either "version 3.8.2+" or "version 3.8.2 or higher", the latter will be understood by more people.